### PR TITLE
Feature/update copyright year

### PR DESCRIPTION
**Pull request by KieranFrance**
Just need a quick sanity check on this.

Caught up in this change is a fix for the settings validation.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -122,8 +122,6 @@
         </profile>
     </profiles>
 
-    <build>
-        <plugins>
```
Comment (daniel-warren) [inline pom.xml:126]: @KieranFrance was this an intentional deletion?

**Review by daniel-warren — CHANGES_REQUESTED** [review]

File: src/main/java/com/idrsolutions/microservice/utils/DefaultFileServlet.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 IDRSolutions
+ * Copyright 2022 IDRsolutions
  * Copyright 2021 OmniFaces
  *
```
Comment (KieranFrance) [inline src/main/java/com/idrsolutions/microservice/utils/DefaultFileServlet.java:4]: @leonatherton Do we need to update the OmniFaces year as well for does this need to remain at 2021?

Comment (leonatherton) [reply src/main/java/com/idrsolutions/microservice/utils/DefaultFileServlet.java:4]: @KieranFrance I would leave it as-is so as to not modify the original copyright notice.
https://fossa.com/blog/open-source-licenses-101-apache-license-2-0/

File: src/main/java/com/idrsolutions/microservice/utils/SettingsValidator.java [hunk 3]
```diff
@@ -76,9 +96,14 @@ public Float validateFloat(final String setting, final float[] range, final bool
...
-                final float fValue = Float.parseFloat(value);
-                if (fValue < range[0] && range[1] < fValue) {
+            if (!value.isEmpty()) {
+                final float fValue;
+                if (value.matches("-?\\d+\\.?(\\d+)?")) {
+                    fValue = Float.parseFloat(value);
+                } else {
+                    fValue = Float.NaN;
+                }
+                if (Float.isNaN(fValue) || (fValue < range[0] || range[1] < fValue)) {
```
Comment (leonatherton) [inline src/main/java/com/idrsolutions/microservice/utils/SettingsValidator.java:106]: @KieranFrance did you intend to change this?

**Review by leonatherton — CHANGES_REQUESTED** [review]

File: src/main/java/com/idrsolutions/microservice/utils/SettingsValidator.java (hunk 3 above)
Comment (KieranFrance) [reply src/main/java/com/idrsolutions/microservice/utils/SettingsValidator.java:106]: I did, it was accidentally caught up in this commit.

**Pull request merged**